src/components/List.jsx

```diff
@@ -0,0 +1,67 @@
+import { AiTwotoneDelete, AiTwotoneEdit } from "react-icons/ai";
+import { useContext } from "react";
+import { TiTick } from "react-icons/ti";
+import TodoContext from "../context/TodoContext";
+import { useState } from "react";
+
+const List = () => {
+  const { allTodos, handleDelete, todoCheck, setAllTodos } =
+    useContext(TodoContext);
+  const [isClicked, setIsClicked] = useState(false);
+  const [editInput, setEditInput] = useState("");
+
+  const handleEditInput = (event) => {
+    setEditInput(event.target.value);
+  };
```

Comment:
Since the functionality in this function fit on one line, you can remove the curly braces.